app/src/main/res/layout/item_overdue_list.xml

```diff
@@ -6,65 +6,126 @@
+        android:background="?android:attr/selectableItemBackground"
+        android:paddingEnd="@dimen/spacing_16"
+        android:paddingStart="@dimen/spacing_16"
+        android:src="@drawable/ic_call"
+        tools:ignore="ContentDescription" />
+    </LinearLayout>
+
+    <LinearLayout
+      android:id="@+id/overdue_bottom_layout"
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:layout_below="@+id/patient_details"
       android:orientation="vertical"
-      android:padding="@dimen/spacing_16">
+      android:paddingBottom="@dimen/spacing_8"
+      android:visibility="gone">
+
+      <View
+        android:layout_width="match_parent"
+        android:layout_height="1dp"
+        android:layout_marginBottom="@dimen/spacing_8"
+        android:background="@color/black_opacity_10" />
 
       <TextView
-        android:id="@+id/overdue_patient_name_gender_age"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/spacing_4"
-        android:textColor="@color/black_opacity_90"
-        android:textSize="@dimen/textsize_16"
-        tools:targetApi="lollipop"
-        tools:text="Anisha Acharya" />
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:paddingBottom="@dimen/spacing_8"
+        android:paddingEnd="@dimen/spacing_16"
+        android:paddingStart="@dimen/spacing_16"
+        android:paddingTop="@dimen/spacing_8"
+        android:text="Show phone number"
+        tools:ignore="HardcodedText" />
 
       <TextView
-        android:id="@+id/overdue_patient_bp"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:layout_marginBottom="@dimen/spacing_4"
-        android:textColor="@color/black_opacity_50"
-        android:textSize="@dimen/textsize_14"
-        tools:text="30 days ago: 175/85" />
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
+        android:paddingBottom="@dimen/spacing_8"
+        android:paddingEnd="@dimen/spacing_16"
+        android:paddingStart="@dimen/spacing_16"
+        android:paddingTop="@dimen/spacing_8"
+        android:text="See patient record"
+        tools:ignore="HardcodedText" />
 
       <TextView
-        android:id="@+id/overdue_days"
-        android:layout_width="wrap_content"
+        android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:textColor="@color/red_500"
-        android:textSize="@dimen/textsize_14"
-        tools:text="4 day overdue" />
+        android:background="?attr/selectableItemBackground"
+        android:clickable="true"
+        android:focusable="true"
```

Comment:
Don't forget to remove `clickable=true` and `focusable=true` once you add item click listeners.

Reply:
Actually, we will forget this. Let's just remove it now?